docs/releases/1.11.txt

```diff
@@ -305,6 +305,10 @@ Database backend API
 * To enable ``FOR UPDATE SKIP LOCKED`` support, set
   ``DatabaseFeatures.has_select_for_update_skip_locked = True``.
 
+* The  ``DatabaseFeatures.can_introspect_index_type`` flag was added. Set it to
```

Comment:
I don't think this needs to be documented right now, since the flag is only used in a test that's essentially PostgreSQL specific at this point. Thoughts?